api/src/pcapi/routes/backoffice/chronicles/blueprint.py

```diff
@@ -0,0 +1,151 @@
+from pcapi.routes.backoffice.utils import permission_required
+from pcapi.utils import string as string_utils
+
+from . import forms
+
+
+chronicles_blueprint = utils.child_backoffice_blueprint(
+    "chronicles",
+    __name__,
+    url_prefix="/chronicles/",
+    permission=perm_models.Permissions.READ_CHRONICLE,
+)
+
+
+@chronicles_blueprint.route("", methods=["GET"])
+@atomic()
+def list_chronicles() -> utils.BackofficeResponse:
+
+    if not FeatureToggle.WIP_ENABLE_CHRONICLES_IN_BO.is_active():
+        raise NotFound()
+
+    form = forms.GetChronicleSearchForm(formdata=utils.get_query_params())
+    if not form.validate():
+        return render_template("chronicles/list.html", rows=[], form=form), 400
+
+    product_subquery = (
+        sa.select(sa.func.array_agg(offers_models.Product.name))
+        .select_from(chronicles_models.ProductChronicle)
+        .join(offers_models.Product, offers_models.Product.id == chronicles_models.ProductChronicle.productId)
+        .filter(chronicles_models.ProductChronicle.chronicleId == chronicles_models.Chronicle.id)
+        .scalar_subquery()
+    )
+
+    query = db.session.query(
+        chronicles_models.Chronicle.id,
+        chronicles_models.Chronicle.content,
+        sa.func.left(chronicles_models.Chronicle.content, 350).label("short_content"),
+        chronicles_models.Chronicle.dateCreated,
+        chronicles_models.Chronicle.isActive,
+        product_subquery.label("products"),
+    )
+    q_filters = []
+    if form.q.data and string_utils.is_ean_valid(form.q.data):
+        query = query.join(chronicles_models.Chronicle.products)
+        q_filters.append(offers_models.Product.extraData["ean"].astext == string_utils.format_ean_or_visa(form.q.data))
+    elif form.q.data:
+        if form.search_type.data in (forms.SearchType.ALL.name, forms.SearchType.CHRONICLE_CONTENT.name):
+            q_filters.append(
+                sa.and_(
+                    chronicles_models.Chronicle.__content_ts_vector__.match(w, postgresql_regconfig="french")
+                    for w in form.q.data.split(" ")
+                )
+            )
+        if form.search_type.data in (forms.SearchType.ALL.name, forms.SearchType.PRODUCT_NAME.name):
+            if form.search_type.data == forms.SearchType.ALL.name:
+                query = query.outerjoin(chronicles_models.Chronicle.products)
+            else:
+                query = query.join(chronicles_models.Chronicle.products)
+            split_product_name = "%".join(form.q.data.split(" "))
+            q_filters.append(offers_models.Product.name.ilike(f"%{split_product_name}%"))
+    if q_filters:
+        query = query.filter(sa.or_(*q_filters))
+
+    if form.date_range.data:
+        if form.date_range.from_date is not None:
+            from_date = datetime.datetime.combine(form.date_range.from_date, datetime.time.min)
+            query = query.filter(chronicles_models.Chronicle.dateCreated > from_date)
```

Comment:
0h00 est inclus dans le jour
```suggestion
            query = query.filter(chronicles_models.Chronicle.dateCreated >= from_date)
```